Sent2Vec Tests and Tutorial.ipynb

```diff
@@ -0,0 +1,3008 @@
+{
+ "cells": [
```

Comment:
Notebook should be in `docs/notebooks` (not root of repo) + please rename it to `sent2vec_tutorial.ipynb`

Reply:
Also, please re-run it with python3